### PR TITLE
Blogpost for unified training operator

### PR DESCRIPTION
**Pull request by johnugeorge**
Draft of the training operator blogpost

### CODE REVIEW

File: _posts/2021-10-13-unified-training-operator-release.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+layout: post
+toc: true
+comments: true
+hide: false
+categories: [release]
+permalink: /unified-training-operator-1.3-release/
+author: "Johnu George, Jiaxin Shan, Josh Bottum"
+---
+
+The Kubeflow Training Operator Working Group introduced several enhancements in the recent Kubeflow 1.4 release. The most significant was the introduction of the new unified [training operator](https://github.com/kubeflow/training-operator) that enables Kubernetes custom resources (CR) for many of the popular training frameworks: Tensorflow, Pytorch, MXNet and XGboost. In addition, the `tf-operatory` repository has been renamed to `training-operator`.
```
Comment (johnugeorge) [inline _posts/2021-10-13-unified-training-operator-release.md:12]: One typo: `tf-operatory` -> `tf-operator`

File: _posts/2021-10-13-unified-training-operator-release.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+* V2beta1 MPIJob API [#366](https://github.com/kubeflow/mpi-operator/pull/366) [#378](https://github.com/kubeflow/mpi-operator/pull/378)
+* Intel MPI Support [#389](https://github.com/kubeflow/mpi-operator/pull/389) [#403](https://github.com/kubeflow/mpi-operator/pull/403) [#417](https://github.com/kubeflow/mpi-operator/pull/417) [#425](https://github.com/kubeflow/mpi-operator/pull/425)
+
+## MPI Operator roadmap
+
+The MPI framework integration with the unified training operator is under development and is planned for delivery in the next release i.e. post 1.4. Currently, it needs to be separately installed using MPIJob [manifests](https://github.com/kubeflow/manifests/tree/v1.4-branch/apps/mpi-job/upstream).
+
+## Acknowledgement
+
+The unified training operator is the outcome of efforts from all existing Kubeflow training operators and aims to provide a unified and simplified experience for both users anddevelopers. We'd like to thank everyone who has contributed to and maintained the original operators.
```
Comment (johnugeorge) [inline _posts/2021-10-13-unified-training-operator-release.md:56]: Space missing: `anddevelopers` -> `and developers`

**Comment by johnugeorge** [comment]
Thanks @theadactyl for putting time on this.
Rest all looks good.
/lgtm

**Comment by zijianjoy** [comment]
/lgtm
/approve

Thank you @johnugeorge for composing this blogpost, and everyone for the review!

**Pull request merged**